### PR TITLE
fix: infra-agent settings in beanstalk docs

### PR DESCRIPTION
**Pull request by josemore**
## Give us some context

* Add a section to the infra agent Beanstalk installation instructions explaining how to configure the agent.

### CODE REVIEW

File: src/content/docs/infrastructure/new-relic-infrastructure/troubleshooting/network-connection-issues.mdx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+Links             : {}
+ParsedHtml        : System.__ComObject
+RawContentLength  : 284
+```
+
+</Collapser>
+
+<Collapser id="linux" title="Linux">
+
+1. **Resolve the domain:** Use nslookup or dig to validate that the Linux system can resolve the domain to an IP address:
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/new-relic-infrastructure/troubleshooting/network-connection-issues.mdx:65]:
```suggestion
1. **Resolve the domain:** Use `nslookup` or `dig` to validate that the Linux system can resolve the domain to an IP address:
```

File: src/content/docs/infrastructure/new-relic-infrastructure/troubleshooting/network-connection-issues.mdx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+3. **Traceroute:** Use traceroute to check the network path between your system and the server:
+
+```
+$ traceroute -I infra-api.newrelic.com
+```
+
+Analyze the output to identify potential network issues, such as high latency or packet loss.
+
+4. **Curl:** Use curl to check if the endpoint is reachable:
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/new-relic-infrastructure/troubleshooting/network-connection-issues.mdx:89]:
```suggestion
4. **Curl:** Use `curl` to check if the endpoint is reachable:
```

File: src/content/docs/infrastructure/new-relic-infrastructure/troubleshooting/network-connection-issues.mdx [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+* Connection #0 to host infra-api.newrelic.com left intact
+```
+
+</Collapser>
+
+</CollapserGroup>
+
+## Troubleshoot with NRDiag
+
+[Code Implementation](https://github.com/newrelic/newrelic-diagnostics-cli/blob/main/tasks/infra/agent/connect.go#L48)
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/new-relic-infrastructure/troubleshooting/network-connection-issues.mdx:119]: @josemore What's the purpose of this link? Can you add something before this link, like what readers would expect to see from this link or why they'd want to go to it?

File: src/content/docs/infrastructure/new-relic-infrastructure/troubleshooting/network-connection-issues.mdx [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+```
+Failure - Infra/Agent/Connect
+
+There was an error connecting to https://log-api.newrelic.com
+Please check network and proxy settings and try again or see -help for more options.
+Error = Get "https://log-api.newrelic.com": dial tcp: lookup log-api.newrelic.com on [::1]:53: read udp [::1]:0->[::1]:53: i/o timeout
+See https://docs.newrelic.com/docs/new-relic-solutions/get-started/networks/#infrastructure for more information.
+```
+
+Note that the nrdiag Connect task result is shown in stdout, it should be collected with the `nrdiag_output.zip` file.
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/new-relic-infrastructure/troubleshooting/network-connection-issues.mdx:152]:
```suggestion
Note that the `nrdiag Connect` task result is shown in `stdout`, it should be collected with the `nrdiag_output.zip` file.
```

**Review by ally-sassman — CHANGES_REQUESTED** [review]
See my comments

**Comment by josemore** [comment]
thanks @ally-sassman, I removed the "network troubleshooting" doc, I'll submit that one in a different PR and already adapted based on your feedback.

**Review by ally-sassman — APPROVED** [review]

**Pull request merged**